cardinal_service_profile.info.yml

```diff
@@ -8,7 +8,6 @@ install:
   - 'address:address'
   - 'admin_toolbar:admin_toolbar'
   - 'admin_toolbar:admin_toolbar_tools'
-  - 'allowed_formats:allowed_formats'
```

Comment:
It doesn't look like this is getting uninstalled -- it's still in the `core.extension.yml`. You'll also want to make sure any allowed format configuration in the third party settings has been migrated to use core allowed formats for any cardinal service specific configuration.

I had to clean these up manually for SDSS. See commits:

- https://github.com/SU-SWS/ace-sdssgryphon/pull/566/commits/134cb612aba2fa23a7c833bc9436aa46d55edea1
- https://github.com/SU-SWS/ace-sdssgryphon/pull/566/commits/333521928bcbda78543ac516d483633a1bdb1c11